mentat/llm_api.py

```diff
@@ -176,8 +209,31 @@ def model_context_size(model: str) -> Optional[int]:
         return None
 
 
-def model_price_per_1000_tokens(model: str) -> Optional[tuple[float, float]]:
+def maximum_context_size() -> int:
```

Comment:
I think it would make sense for this function to also take a model argument (instead of using session context) since it's very standalone